src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -189,7 +194,8 @@ void VulnerabilityScannerFacade::start(
             std::make_unique<RouterSubscriber>("deltas-syscollector", "vulnerability_scanner_deltas");
         m_syscollectorDeltasSubscription->subscribe(
             // coverity[copy_constructor_call]
-            [scanOrchestrator, delayedDispatcher = m_delayedDispatcher](const std::vector<char>& message)
+            [&getSecondsFromEpoch, scanOrchestrator, delayedDispatcher = m_delayedDispatcher](
```

Comment:
Capture getSecondsFromEpoch by copy, because the lifetime of the std::function getSecondsFromEpoch is closed to the start method.

Reply:
Done